lib/kafka_ex.ex

```diff
@@ -137,7 +137,7 @@ defmodule KafkaEx do
 
   Optional arguments(KeywordList)
   - offset: When supplied the fetch would start from this offset, otherwise would start from the last committed offset of the consumer_group the worker belongs to. For Kafka < 0.8.2 you should explicitly specify this.
-  - worker_name: the worker we want to run this fetch request through. Default is KafkaEx.Server
+  - worker_name: the worker we want to run this fetch request through. Default is KafkaEx.server
```

Comment:
This probably warrants some explanation.  Even if it's just "See [some other part of the doc where we explain what KafkaEx.server is]"